### DISCUSSION TITLE
Release Notes 1.15 - CovPass and CovPassCheck

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-android#168] [Announcements]
📢 ***Release Notes CovPass and CovPassCheck Version 1.15*** 📢   
  
**Main Features CovPass:**

•	**Misuse-Prevention**: The max number of people that can store certificates is now limited to 20 people. Furthermore, the app now displays an extended warning when scanning the second person. It includes a link to the store entry of the CovPassCheck. This warning will also appear after 50% of the max capacity is reached.  
   
•	**Validation-as-a-Service**: For booking purposes (e.g. flight and hotel) you can now proof your current vaccination / recovery status to the requesting service partner. The certificates will be validated by a selection trusted validation services within Germany. Those validation services will not store any personal information. No information will be exchanged without the user’s consent. For more transparency additional features will extend the capabilities of this feature in the upcoming releases.  
  
**Planned Main Features CovPassCheck:**
•	No new main features are included in this version. A requested adaption for 2G+ concepts etc. is planned for the release 1.17. More information will be released soon!  
  
**Bug-Fixes:**
•	Multiple minor UI-Bugs will be fixed.   
  
❗ **Attention:**  Please also check the new announcement “Information about the upcoming version 1.16” for further information about upcoming features and bugfixes.

Thank you all for your feedback, ideas and critics!   

stay safe and best regards,

Team CovPass and CovPassCheck ✔️

**Comment by Ein-Tim** [comment]
Appreciate this @alexcimander, thank you!
